cnd/src/http_api/routes/rfc003/handlers/action.rs

```diff
@@ -150,7 +164,7 @@ trait SelectAction<Accept, Decline, Deploy, Fund, Redeem, Refund>:
 }
 
 fn rfc003_accept_response<AL: rfc003::Ledger, BL: rfc003::Ledger>(
```

Comment:
Being in `rfc003/handlers/action.rs` couldn't this method just be like this?
```suggestion
fn accept_response<AL: Ledger, BL: Ledger>(
```
Same goes for any other fully qualified imports here that reference RFC003.